src/renderers/webgpu/utils/WebGPUTextureUtils.js

```diff
@@ -455,6 +460,27 @@ class WebGPUTextureUtils {
 
 	}
 
+	_getDefaultVideoFrame() {
+
+		let defaultVideoFrame = this.defaultVideoFrame;
+
+		if ( defaultVideoFrame === null ) {
+
+			const init = {
+				timestamp: 0,
+				codedWidth: 1,
+				codedHeight: 1,
+				format: 'RGBA',
+			};
+
+		  this.defaultVideoFrame = defaultVideoFrame = new VideoFrame( new Uint8Array( [ 0, 0, 0, 0xff ] ), init );
```

Comment:
Wow, that works!? I wasn't aware that you can manually create instances of `VideoFrame`.

Reply:
TIL: 

> A second constructor enables the creation of a VideoFrame from its binary pixel representation in an [ArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer), a [TypedArray](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray), or a [DataView](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView).